tests/Console/Command/DescribeNameNotFoundExceptionTest.php

```diff
@@ -26,16 +26,6 @@
  */
 final class DescribeNameNotFoundExceptionTest extends TestCase
 {
-    public function testIsInvalidArgumentException(): void
-    {
-        $exception = new DescribeNameNotFoundException(
-            'Peter',
-            'weird'
-        );
-
-        self::assertInstanceOf(\InvalidArgumentException::class, $exception);
```

Comment:
no clue why we have tests like those.
I believe they cause no harm, but also no huge benefit behind them.

IE, we rely that result of `function foo(): Date` is Date without making utest for result type